helper/restore_helper.go

```diff
@@ -280,6 +307,14 @@ func doRestoreAgent() error {
 					if *onErrorContinue && utils.FileExists(fmt.Sprintf("%s_skip_%d", *pipeFile, tableOid)) {
 						logWarn(fmt.Sprintf("Oid %d, Batch %d: Skip file discovered, skipping this relation.", tableOid, batchNum))
 						err = nil
+						skipOid = tableOid
+						/* Close up to *copyQueue files with this tableOid */
+						for idx := 0; idx < *copyQueue; idx++ {
+							batchToDelete := batchNum + idx
+							if batchToDelete < batches {
```

Comment:
could You combine loop and condition like
```go
for batchToDelete := batchNum; batchToDelete < batches && (some other condition); batchToDelete++ {
```

Reply:
I believe my variant is much simpler to read - loop is going over queue size. And don't have any limit for line count and temporary variables - no need for optimizing them